src/Microsoft.ML.PCA/PcaTrainer.cs

```diff
@@ -469,6 +470,20 @@ public void SaveAsText(TextWriter writer, RoleMappedSchema schema)
             }
         }
 
+        public IDataView GetSummaryDataView(RoleMappedSchema schema)
+        {
+            var bldr = new ArrayDataViewBuilder(Host);
+
+            bldr.AddColumn("Mean vector", NumberType.R4, _mean);
+            bldr.AddColumn("Projected mean vector", NumberType.R4, _meanProjected);
+            for (var i = 0; i < _rank; ++i)
+            {
```

Comment:
For single line statements like this, prefer this:

```csharp
for (int i=0; I<foo; ++i)
    yodawg += i;
```

to this

```csharp
for (int i=0; I<foo; ++i)
{
    yodawg += i;
}
```

Those extra `{`s are evil.